src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -54,10 +54,7 @@
   ([sqs-client name]
    (create-queue sqs-client name {}))
 
-  ([sqs-client name
-    {:keys [kms-master-key-id
-            kms-data-key-reuse-period]
-     :as   opts}]
+  ([sqs-client name opts]
```

Comment:
I would leave this for the signature because it tells the user what keys `opts` take.